docs/source/profiler.rst

```diff
@@ -803,6 +803,8 @@ Below is an breakdown of all the options.
       * is_enabled - (Boolean) Enables or disables performing correlation profiling
       * columns - Columns considered to calculate correlation
     * **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
+      * unique_count - (Boolean) Option to enable/disable functionalities for unique row count
```

Comment:
I think we need to clarify the extra options for `unique_count` like `hashing_methods` and other props for `HLL`

Reply:
yeah the nesting should be added for HLL too you'll include `seed` and the other option for HLL